client/my-sites/marketing/tools/google-adwords.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import page from 'page';
+import PropTypes from 'prop-types';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import Button from 'components/button';
+import { getSelectedSite } from 'state/ui/selectors';
+import GoogleVoucherDetails from 'my-sites/checkout/checkout-thank-you/google-voucher';
+import { isPremium, isBusiness, isEcommerce, isEnterprise } from 'lib/products-values';
+import isSiteAtomic from 'state/selectors/is-site-automated-transfer';
+import MarketingToolsFeature from './feature';
+import QuerySiteVouchers from 'components/data/query-site-vouchers';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface ConnectProps {
+	isAtomic: boolean;
+	isJetpack: boolean;
+	isPremiumOrHigher: boolean;
+	siteId: number | null;
+	siteSlug: string | null;
+}
+
+export const GoogleAdwordsCard: FunctionComponent< ConnectProps > = ( {
+	isAtomic,
+	isJetpack,
+	isPremiumOrHigher,
+	siteId,
+	siteSlug,
+} ) => {
+	const translate = useTranslate();
+	if ( ( isJetpack && ! isAtomic ) || ! siteId || ! siteSlug ) {
+		return null;
+	}
+
+	const handleUpgradeClick = () => {
+		page( addQueryArgs( { plan: 'PLAN_PREMIUM' }, `/plans/${ siteSlug }` ) );
+	};
+
+	const renderButton = () => {
+		if ( isPremiumOrHigher ) {
+			return <GoogleVoucherDetails />;
```

Comment:
Note we only bump Google Analytics stats in the [`GoogleVoucherDetails` component](https://github.com/Automattic/wp-calypso/blob/d4ce050d6b2a134c55e6b263427b2ec010c8b974/client/my-sites/checkout/checkout-thank-you/google-voucher/index.jsx#L80). I suspect we should update it to record Tracks events, especially if we want to set up Tracks funnels.

Reply:
Done